building/devices/hikey960.rst

```diff
@@ -30,6 +32,24 @@ Build instructions
 Just follow the instructions at ":ref:`get_and_build_the_solution`". If ``make
 flash`` doesn't work, try ``make recovery``.
 
+For the 4GB RAM board version (rev B), an update to the ``CFG_DRAM_SIZE_GB`` 
+setting in `conf.mk`_ is needed. See the following patch:
```

Comment:
Or simply `make PLATFORM=hikey-hikey960 CFG_DRAM_SIZE_GB=4".

Given that we now have three "flavors" which differ in UART and RAM, perhaps we should distinguish the flavors ? For instance: hikey960v1, hikey960 (that would be v2 rev A which is the target of the default build currently), hikey960v2revB.

Reply:
I also believe that at a separate build target would be the preferred solution here.